wdl/Module07XfBatchEffect.wdl

```diff
@@ -261,49 +231,49 @@ task GetFreqTable {
     RuntimeAttr? runtime_attr_override
   }
   RuntimeAttr default_attr = object {
-    cpu_cores: 1, 
-    mem_gb: 6,
-    disk_gb: 50,
-    boot_disk_gb: 10,
-    preemptible_tries: 3,
-    max_retries: 1
-  }
+                               cpu_cores: 1,
+                               mem_gb: 6,
+                               disk_gb: 50,
+                               boot_disk_gb: 10,
+                               preemptible_tries: 3,
+                               max_retries: 1
+                             }
   RuntimeAttr runtime_attr = select_first([runtime_attr_override, default_attr])
   command <<<
     set -euo pipefail
     #Run vcf2bed
     svtk vcf2bed \
-      --info ALL \
-      --no-samples \
-      ~{vcf} "~{prefix}.vcf2bed.bed"
+    --info ALL \
```

Comment:
I appreciate you taking this opportunity to clean up some of the WDL indentation. However, the indentation in these bash sections is for readability - ie. these lines are indented because they are a continuation of the svtk command, and lines 252-257 are indented further because they are a continuation of the command substitution. Let's keep the bash indentation as it was to help future readers